R/tm_a_pca.R

```diff
@@ -938,6 +946,20 @@ srv_a_pca <- function(id, data, reporter, filter_panel_api, dat, plot_height, pl
       )
     })
 
+    output$all_plots <- renderUI({
+      teal::validate_inputs_segregated(list("Some inputs require attention" = iv_r(),
+                                            "Plot settings are required" = iv_extra))
+      validation()
+      tags$div(
+        class = "overflow-scroll",
+        uiOutput(session$ns("tbl_importance_ui")),
+        hr(),
+        uiOutput(session$ns("tbl_eigenvector_ui")),
+        hr(),
+        teal.widgets::plot_with_settings_ui(id = session$ns("pca_plot"))
+      )
```

Comment:
this should stay in the ui section.

Reply:
Then how duplication of validation messages is avoided.